EmpireLegends/MainGame.cpp

```diff
@@ -40,3 +42,59 @@ void MainGame::afterAction()
 	this->players->rotate();
 	std::cout << getCurrentPlayer()->getName() << " plays next." << std::endl;
 }
+
+void MainGame::maingameloop(vector<Card*> cardDeck, vector<Player*> myPlayers) {
+
+    int cardposition = 0;
+    while (myPlayers[0]->getNumofcards() != 13|| myPlayers[1]->getNumofcards() != 13) {
```

Comment:
The condition should be that they are both at 13. The game ends when all players have the max amount. 

Also, I know for this assignment it says it's for a game of 2 players, but the main loop should use what's created in the game start part, hence it should also work for other game sizes and any set max amount